sys-apps/lshw/lshw-02.19.2b_p20210121-r4.ebuild

```diff
@@ -65,6 +65,11 @@ src_prepare() {
 src_compile() {
 	tc-export CC CXX AR PKG_CONFIG
 	use static && append-ldflags -static
+	# Some toolchains are defaulting to C++17, which causes
+	# `<sys-apps/lshw-02.19.2b_p20220831` to break due to its use of the
+	# `register` keyword. Just pin it at 14, since future versions don't
+	# have this issue.
+	append-cxxflags '-std=c++14'
```

Comment:
not a new issue, but all of this logic really belongs in `src_configure`.  should move that at some point.